frontend/public/src/containers/pages/DashboardPage.js

```diff
@@ -31,19 +22,18 @@ import {
   courseEmailsSubscriptionMutation
 } from "../../lib/queries/enrollment"
 import { currentUserSelector } from "../../lib/queries/users"
-import {
-  isSuccessResponse,
-  isProgramUIEnabled,
-} from "../../lib/util"
-import { routes } from "../../lib/urls"
+import { isProgramUIEnabled } from "../../lib/util"
 import { addUserNotification } from "../../actions"
-import { EnrolledItemCard } from "../../components/EnrolledItemCard"
-import { ProgramEnrollmentCard } from "../../components/ProgramEnrollmentCard"
```

Comment:
Can we get rid of this component now that it is not used anywhere?

Reply:
Culled in https://github.com/mitodl/mitxonline/pull/1090/commits/76ce83b9ef83c3e2d11829f9e61e487268ce651b